main/webapp/modules/core/langs/translation-en.json

```
@@ -778,6 +778,7 @@
     "core-views/use-values-as-identifiers/header": "Use values as identifiers",
     "core-views/use-values-as-identifiers2": "Mark cells as reconciled with their values as identifiers",
     "core-views/choose-reconciliation-service": "Choose a reconciliation service",
+    "core-views/entity URL":"Entity URL",
```

For consistency with other message keys:
```suggestion
    "core-views/entity-URL":"Entity URL",
```